### PR TITLE
Usecentrics

### PR DESCRIPTION
**Pull request by chrmod**

### CODE REVIEW

File: db/organizations/usercentrics.eno [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: Usercentrics
+website_url: https://usercentrics.com/
+privacy_policy_url: https://usercentrics.com/privacy-policy/
+privacy_contact: privacy@usercentrics.com
+country: DE
+description: Consent Managment Platform founded in 2012
```
Comment (philipp-classen) [inline db/organizations/usercentrics.eno:6]: Has a typo. But the company description would benefit from some improvements.

Comment (chrmod) [inline db/organizations/usercentrics.eno:6]:
```suggestion
description: Usercentrics solution enables companies to collect, manage and document user consent on all digital channels, such as websites or apps
```

Comment (philipp-classen) [reply db/organizations/usercentrics.eno:6]: What about:
```
Usercentrics GmbH, founded in 2012, offers Consent Management Platforms (CMP) that enables the integration of cookies into websites or mobile apps and the management of user consents (opt-ins). The software includes monitoring, analytics, and configuration features, making it adaptable to diverse usage scenarios.
```

Comment (chrmod) [inline db/organizations/usercentrics.eno:6]:
```suggestion
description: Usercentrics GmbH, founded in 2012, offers Consent Management Platforms (CMP) that enables the integration of cookies into websites or mobile apps and the management of user consents (opt-ins). The software includes monitoring, analytics, and configuration features, making it adaptable to diverse usage scenarios.
```

**Pull request merged**